### PR TITLE
Set GKE environment variables properly.

### PR DESCRIPTION
**Pull request by spxtr**

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml [hunk 1]
```diff
@@ -210,7 +209,11 @@
 
               # GKE variables
               export CLUSTER_NAME=${{E2E_NAME}}
+              export ZONE="us-central1-f"
               export KUBE_GKE_NETWORK=${{E2E_NAME}}
+              export CLOUDSDK_BUCKET="gs://cloud-sdk-testing/ci/staging"
```
Comment (ixdy) [inline jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml]: is this one necessary? or just cargo-culted from other configs?

Comment (spxtr) [inline jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml]: No idea, cargo-culted from kubernetes-e2e-gke.

Comment (ixdy) [inline jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml]: The comment was lost in #124, but it looks OK:

```yaml
        # By default, GKE tests run against the GKE test endpoint using CI Cloud SDK.		
        # Release jobs (e.g. prod, staging, and test) override these two variables.		
        export CLOUDSDK_BUCKET="gs://cloud-sdk-testing/ci/staging"		
        export CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER="https://test-container.sandbox.googleapis.com/"		
```

**Comment by ixdy** [comment]
lgtm

**Pull request merged**